qiskit/transpiler/passes/utils/check_map.py

```diff
@@ -65,10 +86,12 @@ def run(self, dag):
         Args:
```

Comment:
I can't comment on the docstring above, but it would be better if it is something like this: 
```text
        If `dag` is mapped to `coupling_map`, the property whose name is specified
        in `self.property_set_field` is set to True (or to False otherwise).
```
since with the newer than `property_set_field` than this docstring the property field might be named differently.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/14925